.env.example

```diff
@@ -1 +1,25 @@
-PORT=5000
\ No newline at end of file
+PORT=5000
+
+## Production
+DB_NAME = ''
+DB_USERNAME = ''
+DB_PASSWORD = ''
+DB_HOST = ''
+DB_PORT = ''
+DATABASE_URL = ''
+
+## local
+LOCAL_DB_NAME = ''
+LOCAL_DB_USERNAME = ''
+LOCAL_DB_PASSWORD = ''
+LOCAL_DB_HOST = ''
+LOCAL_DB_PORT = ''
+LOCAL_DB_URL = ''
+
+## test
+CI_DB_NAME = ''
```

Comment:
Are you this is how we should name env variable, please update this file to have the same variable you are using in normal code.